certora/README.md

```diff
@@ -19,18 +21,26 @@ Please ensure that `CERTORAKEY` is set up in your environment.
 The Bundler3 contract enables an EOA to call different endpoint contracts onchain as well as grouping several calls in a single bundle.
 These calls may themselves reenter the Bundler3 contract.
 
-### Folders and file structure
+## Bundler3 safety restrictions
 
-The [`certora/specs`](specs) folder contains these files:
+A key feature of the bundler is to restrict reentering in adapter calls.
+That is we check that reentering an adapter is only possible during a multicall execution.
+We also check that adapters' methods used during a bundle execution may only be called by the Bundler3 contract.
+We check that it's only possible to reenter the adapter using a dedicated Bunlder3 function.
```

Comment:
```suggestion
In particular, it is checked that reentering an adapter is only possible during a multicall execution.
It is also checked that adapters' methods used during a bundle execution may only be called by the Bundler3 contract.
Additionally, the verification makes sure that it's only possible to reenter the adapter using a dedicated Bundler3 function.
```
let's try to avoid the word "we" in general